paimon-web-server/src/main/java/org/apache/paimon/web/server/data/dto/CatalogDto.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.server.data.dto;
+
+import lombok.Data;
+
+@Data
+public class CatalogDto {
+    private String catalogType;
```

Comment:
Does the DTO need catalog id?

Reply:
It's only a DTO. need't id.

Reply:
@hunter-cloud09, why not DTO need id? Any standard?

Reply:
The id is automatically generated by the snowflake model, so there is no need to pass in the id when creating the catalog.